sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -45,4 +45,37 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_command_setBurstMode</EFDB_Topic>
+    <item>
+      <EFDB_Name>value</EFDB_Name>
+      <Description>Attribute required by the SAL API, but is unused.</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
```

Comment:
Topics need not have any public fields (at one time they did, but that was years ago). Please delete this and all the others like it.